resources/building-mobile-extensions/interacting-with-rules.md

```diff
@@ -86,13 +82,13 @@ You should provide support for the events, actions, and conditions that you publ
 
 ### Publishing a shared state `condition` at runtime
 
-A shared state that was published by your extension can be used as a `condition` when configuring a rule in Experience Platform Launch. To learn how you can publish a shared state, see [Updating the shared state](https://aep-sdks.gitbook.io/docs/resources/building-mobile-extensions/updating-the-shared-state).
+A shared state that was published by your extension can be used as a `condition` when configuring a rule in Experience Platform Launch. To learn how you can publish a shared state, see the [updating the shared state document](https://aep-sdks.gitbook.io/docs/resources/building-mobile-extensions/updating-the-shared-state).
 
 ### Dispatching an event `condition` at runtime
 
-An event that was dispatched by your extension can be used as a `condition` when configuring a rule in Experience Platform Launch. To learn how you can dispatch an event, see [Dispatching events from your extension](https://aep-sdks.gitbook.io/docs/resources/building-mobile-extensions/dispatching-events-from-your-extension).
+An event that was dispatched by your extension can be used as a `condition` when configuring a rule in Experience Platform Launch. To learn how you can dispatch an event, see the [dispatching events from your extension document](https://aep-sdks.gitbook.io/docs/resources/building-mobile-extensions/dispatching-events-from-your-extension).
 
 ### Handling an event `action` at runtime
 
-For an event that your extension registered, a listener for can be used as an `action` when configuring a rule in Experience Platform Launch. To learn how you can register a listener for your events, see [Listening for events](https://aep-sdks.gitbook.io/docs/resources/building-mobile-extensions/event-listeners).
+For an event that your extension registered, a listener for can be used as an `action` when configuring a rule in Experience Platform Launch. To learn how you can register a listener for your events, see the [listening for events document](https://aep-sdks.gitbook.io/docs/resources/building-mobile-extensions/event-listeners).
 
```

Comment:
Not sure why the several sentences above are highlighted in red? Sometimes due to a syntax error.

Reply:
I'm not sure why either, but I did notice that the code blocks above had some incorrectly formatted JSON - so maybe that's why? Not sure.